README.md

```diff
@@ -571,74 +560,86 @@ contacted drand node. If the encryption is not correct, the command outputs an
 error instead.
 
 ## DrandJS
-
-To facilitate the use of drand's randomness in JavaScript-based applications, we
-provide [DrandJS](https://github.com/PizzaWhisperer/drandjs). The main method
-`fetchAndVerify` of this JavaScript library fetches from a drand node the latest
-random beacon generated and then verifies it against the distributed key.  For
-more details on the procedure and instructions on how to use it, refer to the
+To facilitate the use of drand's randomness in JavaScript-based applications,
+we provide [DrandJS](https://github.com/PizzaWhisperer/drandjs). The main
```

Comment:
New url is https://github.com/drand/drandjs

Reply:
Updated.